BankingSystem/BankingSystem/Account.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Bank
+{
+    class Account
+    {
+        public string AccountName { get; private set; }
```

Comment:
No need to add "Account" before name of every field. They are part of class named Account